packages/flutter_test/lib/src/matchers.dart

```diff
@@ -592,7 +592,19 @@ class _HasGoodToStringDeep extends Matcher {
 /// sets of value for which a metric space is defined.
 typedef num DistanceFunction<T>(T a, T b);
 
-const Map<Type, DistanceFunction<dynamic>> _kStandardDistanceFunctions = const <Type, DistanceFunction<dynamic>>{
+/// The type of a union of instances of DistanceFunction<T> for various types T.
```

Comment:
@Hixie This is the change we discussed when I was last in MTV.  Let me know if you have better suggestions for the comment wording, or would prefer to mediate through this through a generic `DistanceFunction<T> getDistanceFunction<T>(Type t)` API.